dev-python/python-msrplib/python-msrplib-0.18.0-r1.ebuild

```diff
@@ -21,6 +21,11 @@ RDEPEND="
 	dev-python/python-application[${PYTHON_USEDEP}]
 	dev-python/python-eventlib[${PYTHON_USEDEP}]
 	dev-python/python-gnutls[${PYTHON_USEDEP}]
-	dev-python/twisted-core[${PYTHON_USEDEP}]
-	dev-python/twisted-names[${PYTHON_USEDEP}]
+	dev-python/twisted[${PYTHON_USEDEP}]
 "
+
+python_test() {
+        pushd test > /dev/null
```

Comment:
All external commands need `|| die` (i.e. those not defined by the package manager or eclasses).